### DISCUSSION TITLE
Problema installazione hacs

### DISCUSSION THREAD

**Discussion by laurafabry** [indomus/forum#720] [HUB: Home Assistant]
ciao a tutti ho un problem con la installazione di Hacs.

inserito la stringa per il token nel file configuration.yaml e messo all'interno della cartella config la cartella custom_component con all'interno la cartella hacs coi file scaricati su github.
quando vado a fare la convalida della configurazione mi da questo errore:
Component error: hacs - No module named 'custom_components.hacs.tasks'
di conseguenza non riesco a riavviare , ho provato a vedere nel menu impostazioni/integrazioni e se faccio aggiungi integrazione trovo il l'integrazione Hacs ma se clicco sopra mi dice "impossibile caricare flusso di integrazione"

qualcuno puo' darmi una mano?

saluti
Fabrizio

**Comment by Scialla** [comment]
Hai seguito la guida https://indomus.it/guide/installare-hacs-home-assistant-community-store-sul-proprio-hub/ ?
Prova a fare la configurazione via Lovelace SENZA il token, come descritto in guida

**Reply by laurafabry** [reply]
Ciao e grazie della risposta , si ho seguito quella guida  che dici tu ma nulla....
Se faccio con Lovelace togliendo il token e la cartella custom component quando vado su aggiungi integrazione non trovo HACS

**Reply by Scialla** [reply]
La guida funziona, hai quindi probabilmente sbagliato qualcosa. Ti consiglio di ricominciare dall'inizio ed installare nuovamente il component, se non va con la procedura manuale prova quella da terminale

**Reply by laurafabry** [reply]
Stasera provo a verificare e ti faccio sapere
